azure-pipelines.yml

```diff
@@ -188,7 +188,7 @@ stages:
         -TsaProjectName "DEVDIV"
         -TsaNotificationEmail "dnceng@microsoft.com"
         -TsaCodebaseAdmin "REDMOND\dn-bot"
-        -TsaBugAreaPath "DevDiv\NET\NET Core "
+        -TsaBugAreaPath "DevDiv\NET Core "
```

Comment:
I wonder if it makes sense to make this a default value that Arcade populates. That way we could control similar errors that happen underneath us by fixing it in Arcade and flowing to all repos. (Not saying we should do that as part of this change, but maybe open an issue to evaluate the feasibility?)

Reply:
Yep opening an issue to evaluate this is a good idea.